docs/index.rst

```diff
@@ -10,53 +10,49 @@
     PHP
     ===
 
-The PHP SDK for Sentry supports PHP 5.3 and higher.  It's
-available as a BSD licensed Open Source library.
+The PHP SDK for Sentry supports PHP 5.6 and higher and is available as a BSD
+licensed Open Source library.
 
 Installation
 ------------
+To install the SDK you will need to be using Composer_ in your project. To install
+it please see the `docs <https://getcomposer.org/download/>`_.
 
-There are various ways to install the PHP integration for Sentry.  The
-recommended way is to use `Composer <http://getcomposer.org/>`__::
+Sentry PHP is not tied to any specific library that sends HTTP messages. Instead,
+it uses Httplug_ to let users choose whichever PSR-7 implementation and HTTP client
+they want to use.
 
-    $ composer require "sentry/sentry"
+If you just want to get started quickly you should run the following command:
 
-Configuration
--------------
+.. code-block:: bash
 
-The most important part is the creation of the raven client.  Create it
-once and reference it from anywhere you want to interface with Sentry:
+    php composer.phar require sentry/sentry php-http/curl-client guzzlehttp/psr7
```

Comment:
I would add an example on how to use Guzzle as a transport too, since it's so popular that a lot of people already use it.